__tests__/test-utils/actions.js

```diff
@@ -0,0 +1,296 @@
+/* globals describe, expect, it */
+
+const deepEqual = (actual, expected) => {
+  if (Array.isArray(actual)) {
+    for (let i = 0; i < actual.length; i++) {
+      deepEqual(actual[i], expected[i])
+    }
+  } else if (actual === null) {
+    expect(actual).toEqual(expected)
+  } else if (typeof actual === 'object') {
+    Object.keys(actual).forEach((k) => {
```

Comment:
I'm surprised there isn't a built in one to Jest. Seems like a lot of people would want this assertion.